test/src/dftbp/api/mm/testers/test_mpic.c

```diff
@@ -0,0 +1,135 @@
+/*------------------------------------------------------------------------------------------------*/
+/*  DFTB+: general package for performing fast atomistic simulations                              */
+/*  Copyright (C) 2006 - 2023  DFTB+ developers group                                             */
+/*                                                                                                */
+/*  See the LICENSE file for terms of usage and distribution.                                     */
+/*------------------------------------------------------------------------------------------------*/
+
+/**
+ * MPI parallel calculation via the API and checks the result
```

Comment:
Thank you for this example!

Reply:
It doesn't do anything with coordinates, but the call prototype is in dftbplus.h